pydicom/config.py

```diff
@@ -262,7 +262,7 @@ def get_overlay_array(ds, group):
 
 Handlers shall have four methods:
 
-def supports_transfer_syntax(ds)
+def supports_transfer_syntax(ds.file_meta.TransferSyntaxUID: UID)
```

Comment:
A dotted name is not a valid function signature.  Should probably be `def supports_transfer_syntax(transfer_syntax: UID)`

Reply:
Thanks for review
I updated the code as your comment 
commit: f51e9579b8b24c45db90c680ffd3741cf5a74769